lib/centurion/mock.rb

```diff
@@ -0,0 +1,49 @@
+module Centurion
+  class Mock
+    def initialize(opts)
+      environment = opts[:current_environment]
+      @options = opts[environment]
+    end
+
+    def run
+      require 'pry-byebug'
```

Comment:
Thanks @frankywahl, I actually needed something recently that this could have been used for.

I'm looks like I'm late to the party, but did you intend to leave the `require 'pry-byebug'` statement here?

```
** Invoke environment:staging (first_time)
** Invoke environment:common (first_time)
** Execute environment:common
** Execute environment:staging
** Invoke centurion:setup (first_time)
** Invoke centurion:clean_environment (first_time)
** Execute centurion:clean_environment
** Execute centurion:setup
/Users/xac/.rbenv/versions/1.9.3-p194/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require': cannot load such file -- pry-byebug (LoadError)
	from /Users/xac/.rbenv/versions/1.9.3-p194/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
	from /Users/xac/newrelic/centurion/lib/centurion/mock.rb:9:in `run'
	from ./bin/centurion:82:in `<main>'
```